resources/lang.js

```diff
@@ -109,69 +109,4 @@ class CactbotLanguage {
     Object.freeze(this.kAbilNameToId);
     Object.freeze(this.kAbilIdToName);
   }
-
-  // Due to this bug: https://github.com/ravahn/FFXIV_ACT_Plugin/issues/100
-  // We can not look for log messages from FFXIV "You use X" here. Instead we
-  // look for the actual ability usage provided by the XIV plugin.
-  // Also, the networked parse info is given much quicker than the lines from the game.
-  youUseAbilityRegex() {
```

Comment:
Can you leave these in CactbotLanguage but remove them from CactbotLanguageEn and CactbotLanguageKo so that they are not duplicated? I think youStartUsingRegex, youGainEffectRegex, youLoseEffectRegex, abilityRegex, gainsEffectRegex, and losesEffectRegex should all be in CactbotLanguage if they are shared.  Sorry if I haven't communicated this clearly enough--that was why I moved these functions from CactbotLanguageEn to CactbotLanguage in master.

The countdown regex functions (since they'll be different) can be in CactbotLanguageEn and CactbotLanguageKo.

Reply:
This is my merging fault, sorry ;( - cherry-picked and amended on [04501a1](https://github.com/hibiyasleep/cactbot/commit/04501a1216b510e02dc45dbc730648a27f212ea5).